docs/3.0-migration-guide.d/remove_support_for_tls_1.0_1.1_and_dtls_1.0.md

```diff
@@ -0,0 +1,11 @@
+Remove suport for TLS 1.0, 1.1 and DLTS 1.0
```

Comment:
Typo: DLTS -> DTLS (I'm glad I'm not alone, LTS releases of Mbed TLS always trip my fingers).

Reply:
:-) Done